lib/App/MoarVM/Debug.rakumod

```diff
@@ -342,11 +355,26 @@ sub caller(Int() $handle --> Nil) {
     say $result.&to-json(:pretty);
 }
 
-sub clearbp(Str() $file, Int() $line --> Nil) {
+multi sub clearbp(Int() $id --> Nil) {
+    say "Breakpoint with this ID ($id) does not exist" and return unless defined %breakpoints{$id};
+    my ($file, $line) = %breakpoints{$id}<file line>;
+    clearbp $file, $line, $id;
+}
+
+multi sub clearbp(Str() $file, Int() $line, Int() $id? is copy --> Nil) {
+    if !defined $id {
+        for %breakpoints.kv -> $k, $v {
+            if $v { if $v<file> eq $file and $v<line> eq $line { $id = $k; last; } }
+        }
+        say "No breakpoint like that ($file:$line) exists" and return if not defined $id
+    }
```

Comment:
I'd probably use something like `$id //= %breakpoints.first({ ... }, :k);` but not exactly sure how I would put the error output in there